terraform/groups/ecs-service/locals.tf

```diff
@@ -4,17 +4,17 @@ locals {
   stack_name                 = "company-requests" # this must match the stack name the service deploys into
   name_prefix                = "${local.stack_name}-${var.environment}"
   global_prefix              = "global-${var.environment}"
-  service_name               = "lfp-pay-web"
+  service_name               = "penalty-payment-web"
```

Comment:
Revert these terraform changes until we've got pipeline updates sorted...see SAN-50.

Reply:
Revert back to lfp-pay-web as suggested